CommonMark/CommonMarkException.cs

```diff
@@ -8,7 +8,7 @@ namespace CommonMark
     /// <summary>
     /// An exception that is caught during CommonMark parsing or formatting.
     /// </summary>
-#if v2_0 || v3_5 || v4_0 || v4_5
+#if v2_0 || v3_5 || v4_0 || (v4_5 && !portable)
```

Comment:
not really, the problem is that the nuget manager might choose the wrong package for 4.5

Adding the whole CSShim as a dependency is not really a good option since it seems way too large. If just the correct class could be added, it would be much more reasonable solution (plus it can be conditionally included only for the portable target).